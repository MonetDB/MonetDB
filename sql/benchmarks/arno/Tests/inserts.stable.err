stderr of test 'inserts` in directory 'sql/benchmarks/arno` itself:


# 14:29:40 >  
# 14:29:40 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=33812" "--set" "mapi_usock=/var/tmp/mtest-483919/.s.monetdb.33812" "--forcemito" "--dbpath=/home/ferreira/repositories/MonetDB-Nov2019/BUILD/var/MonetDB/mTests_sql_benchmarks_arno" "-d0" "--set" "embedded_c=true"
# 14:29:40 >  

# builtin opt 	gdk_dbpath = /home/ferreira/repositories/MonetDB-Nov2019/BUILD/var/monetdb5/dbfarm/demo
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_ipv6 = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 33812
# cmdline opt 	mapi_usock = /var/tmp/mtest-483919/.s.monetdb.33812
# cmdline opt 	gdk_dbpath = /home/ferreira/repositories/MonetDB-Nov2019/BUILD/var/MonetDB/mTests_sql_benchmarks_arno
# cmdline opt 	embedded_c = true

# 14:29:40 >  
# 14:29:40 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-483919" "--port=33812"
# 14:29:40 >  


# 14:29:41 >  
# 14:29:41 >  "Done."
# 14:29:41 >  

