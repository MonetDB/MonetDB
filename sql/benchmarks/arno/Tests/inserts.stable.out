stdout of test 'inserts` in directory 'sql/benchmarks/arno` itself:


# 14:29:40 >  
# 14:29:40 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=33812" "--set" "mapi_usock=/var/tmp/mtest-483919/.s.monetdb.33812" "--forcemito" "--dbpath=/home/ferreira/repositories/MonetDB-Nov2019/BUILD/var/MonetDB/mTests_sql_benchmarks_arno" "-d0" "--set" "embedded_c=true"
# 14:29:40 >  

# MonetDB 5 server v11.35.0 (hg id: 957d7b1a190c)
# This is an unreleased version
# Serving database 'mTests_sql_benchmarks_arno', using 8 threads
# Compiled for x86_64-pc-linux-gnu/64bit with 128bit integers
# Found 15.503 GiB available main-memory.
# Copyright (c) 1993 - July 2008 CWI.
# Copyright (c) August 2008 - 2020 MonetDB B.V., all rights reserved
# Visit https://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://localhost.localdomain:33812/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-483919/.s.monetdb.33812
# MonetDB/GIS module loaded
# MonetDB/SQL module loaded

# 14:29:40 >  
# 14:29:40 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-483919" "--port=33812"
# 14:29:40 >  

#START TRANSACTION;
#COPY 188 RECORDS INTO "sys"."model" FROM stdin USING DELIMITERS E'\t',E'\n','"';
#1	"T"	-1	4
#2	"F"	-1	1
#3	"F"	-1	2
#4	"T"	-2	5
#5	"F"	-2	2
#6	"T"	-3	4
#7	"F"	-1	4
#8	"T"	-1	3
#9	"F"	-1	2
#10	"T"	-1	5
#11	"T"	-1	4
#12	"T"	-2	4
#13	"T"	-2	4
#14	"F"	-1	2
#15	"T"	-2	5
#16	"T"	-2	4
#17	"F"	-1	1
#18	"T"	-2	3
#19	"F"	-1	2
#20	"T"	-2	3
#21	"T"	-2	4
#22	"T"	-2	5
#23	"T"	-2	5
#24	"T"	-2	7
#25	"T"	-1	3
#26	"T"	-2	2
#27	"T"	-2	6
#28	"T"	-2	4
#29	"T"	-3	2
#30	"T"	-2	5
#31	"T"	-2	4
#32	"T"	-3	3
#33	"T"	-2	4
#34	"F"	-1	3
#35	"T"	-1	3
#36	"F"	-1	3
#37	"T"	-1	4
[ 188	]
#COPY 4893 RECORDS INTO "sys"."atom" FROM stdin USING DELIMITERS E'\t',E'\n','"';
#"d1_1"	1	"c"	"22"	0
#"d1_2"	1	"c"	"22"	0
#"d1_3"	1	"c"	"22"	0
#"d1_4"	1	"c"	"195"	0
#"d1_5"	1	"c"	"195"	0
#"d1_6"	1	"c"	"22"	0
#"d1_7"	1	"h"	"3"	0
#"d1_8"	1	"h"	"3"	0
#"d1_9"	1	"h"	"3"	0
#"d1_10"	1	"h"	"3"	0
#"d1_11"	1	"c"	"27"	0
#"d1_12"	1	"c"	"27"	0
#"d1_13"	1	"c"	"22"	0
#"d1_14"	1	"c"	"22"	0
#"d1_15"	1	"h"	"3"	0
#"d1_16"	1	"h"	"3"	0
#"d1_17"	1	"c"	"22"	0
#"d1_18"	1	"c"	"22"	0
#"d1_19"	1	"c"	"22"	0
#"d1_20"	1	"c"	"22"	0
#"d1_21"	1	"h"	"3"	0
#"d1_22"	1	"h"	"3"	0
[ 4893	]
#COPY 5243 RECORDS INTO "sys"."bond" FROM stdin USING DELIMITERS E'\t',E'\n','"';
#0	1	"d1_1"	"d1_2"	"7"
#1	1	"d1_2"	"d1_3"	"7"
#2	1	"d1_3"	"d1_4"	"7"
#3	1	"d1_4"	"d1_5"	"7"
#4	1	"d1_5"	"d1_6"	"7"
#5	1	"d1_6"	"d1_1"	"7"
#6	1	"d1_1"	"d1_7"	"1"
#7	1	"d1_2"	"d1_8"	"1"
#8	1	"d1_3"	"d1_9"	"1"
#9	1	"d1_6"	"d1_10"	"1"
#10	1	"d1_4"	"d1_11"	"7"
#11	1	"d1_11"	"d1_12"	"7"
#12	1	"d1_12"	"d1_13"	"7"
#13	1	"d1_13"	"d1_14"	"7"
#14	1	"d1_14"	"d1_5"	"7"
#15	1	"d1_13"	"d1_15"	"1"
#16	1	"d1_14"	"d1_16"	"1"
#17	1	"d1_11"	"d1_17"	"7"
#18	1	"d1_17"	"d1_18"	"7"
[ 5243	]
#COMMIT;

# 14:29:41 >  
# 14:29:41 >  "Done."
# 14:29:41 >  

